client-api/src/main/java/org/sonarsource/sonarlint/core/client/api/connected/GetSecurityHotspotRequestParams.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * SonarLint Core - Client API
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.sonarlint.core.client.api.connected;
+
+public class GetSecurityHotspotRequestParams {
+  public final String hotspotKey;
```

Comment:
I think on SQ side this is named uuid. Key is reserved for functional identifiers (that the user can see and possibly update).

Reply:
In the WS I use it is named `key`, I don't find any reference to `uuid` (maybe in the internals of SQ ? could you share a pointer if you have something in mind ?). Moreover UUIDs have a specific format, and I don't think the `key` described here would be considered as a UUID (example value: `AXLMPKUw2vtYjVa-RvB5`)

Reply:
I can see only references to `key` in the [APIs for hotspots](https://next.sonarqube.com/sonarqube/web_api/api/hotspots?internal=true)